app/helpers/image_service_helper.rb

```diff
@@ -16,18 +22,34 @@ def riiif_info_url (riiif_file_id)
   # Defaults copied from riiif defaults. https://github.com/curationexperts/riiif/blob/67ff0c49af198ba6afcf66d3db9d3d36a8694023/lib/riiif/routes.rb#L21
   #
   # Returns relative url unless we've defind a riiif server in config/environments/*.rb
-  def riiif_image_url(riiif_file_id, format: 'jpg', size: "full", quality: 'default')
-    path = riiif.image_path(riiif_file_id, locale: nil, size: size, format: format, quality: quality)
-    create_riiif_url(path)
+  def iiif_image_url(image_file_id, format: 'jpg', size: "full", quality: 'default')
+    service = CHF::Env.lookup(:image_server)
+    case service
+    when 'riiif'
+      path = riiif.image_path(image_file_id, locale: nil, size: size, format: format, quality: quality)
+    when 'cantaloupe'
+      path = cantaloupe_image_path(image_file_id, size: size, format: format, quality: quality)
+    end
+    create_iiif_url(path)
+  end
+
+  def cantaloupe_info_path(file_id)
+    "iiif/2/#{CGI.escape(file_id)}/info.json"
+  end
+
+  def cantaloupe_image_path(file_id, size:, format:, quality:)
+    region = 'full'
+    rotation = '0'
```

Comment:
id make 'region' a keyword arg that defaults to `full` instead of hard-coding to full. Maybe rotation too, although we prob have less of a need for it.  I think this is how the riiif helpers basically work, optional args, but you can still supply them. 

Or, I suppose there's an argument for _not_ doing what we just suggested if we don't need it yet.